### PR TITLE
Using React Mutations to update posted data from the server

### PR DESCRIPTION
**Pull request by twentymurial33**
This is going to be my biggest PR ever ...

- Add Form on the AddFood Page - Added the presentation layer 💯 
- Update the schema by adding fields manually -Updated the schema however created a new migration with new keys 👎 
- 
image

- Manage caching by using mutations 

https://redux-toolkit.js.org/rtk-query/usage/mutations using React Mutations 
https://tkdodo.eu/blog/practical-react-query

### CODE REVIEW

File: prisma/schema.prisma [hunk 1]
```diff
@@ -8,8 +8,9 @@ generator client {
...
 model Food {
-    id     Int    @id @default(autoincrement())
-    title  String @unique
-    img    String @unique
-    author String
+    id         Int       @id @default(autoincrement())
+    title      String    @unique
+    img        String    @unique
+    author     String
+    created_at DateTime? @default(now())
```
Comment (brentmclark) [inline prisma/schema.prisma:15]: Yessssss 💪

File: client/src/pages/AddFood.js [hunk 2]
```diff
@@ -1,37 +1,108 @@
...
+    border: 1px solid #ddd;
+    box-sizing: border-box;
+    display: block;
+  }
+
+  Form {
+    max-width: 400px;
+    margin: 0 auto;
+    text-align: center;
+  }
```
Comment (brentmclark) [inline client/src/pages/AddFood.js:105]: I'm assuming this doesn't do anything since `Form` with a capital F isn't a valid HTML element tag.

Comment (twentymurial33) [reply client/src/pages/AddFood.js:105]: It does..it actually styles the Form element...I was surprised too

File: client/src/pages/AddFood.js [hunk 3]
```diff
@@ -1,37 +1,122 @@
...
-    axios
-      .post(url, { title: "title", img: "img", author: "author" })
-      .then((response) => {
-        setFood(response.data);
-      });
-  }, [url]);
+  const mutation = useMutation({
+    mutationFn: (newFood) => {
+      return axios
+        .post(url, { title: "title", img: "img", author: "author" })
```
Comment (brentmclark) [inline client/src/pages/AddFood.js:26]: this line can be changed to:

```suggestion
        .post(url, newFood)
```

**Review by brentmclark — APPROVED** [review]
I assume there's more to come with `yup` and the overall flow of adding new food items.  In its current state it should be OK to merge; otherwise, I'm happy to keep working on this PR for a while until it's all flowing the way you wnt.  Your call!

**Pull request merged**